bin/run-pacta

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+here=$(pwd)
+pacta=${1:-$here}
+parent=$(dirname $pacta)
+
+cp -r "$parent"/input/*.yml "$pacta"/working_dir/10_Parameter_File
+cp -r "$parent"/input/*.csv "$pacta"/working_dir/20_Raw_Inputs
```

Comment:
Note to slef: Since we are just dumping whatever input into the `working_dir/20_Raw_Inputs` folder, and this folder, by default, already contains `TestPortfolio_Input.csv`, the output of running this command will always contain a `TestPortfolio_Input` output.

Reply:
Slef!? Who's slef?!!

Reply:
I am very funny.